egs/ami/s5/local/tfrnnlm/RESULTS

```diff
@@ -0,0 +1,25 @@
+Here are some numbers comparing rescoring performances of different RNNLM toolkits
+
+Un-rescored baseline:
+%WER 24.2 | 13098 94476 | 79.2 12.0 8.8 3.4 24.2 57.4 | -0.157 | exp/ihm/nnet3/tdnn_sp/decode_dev/ascore_12/dev_hires.ctm.filt.sys
+%WER 25.4 | 12643 89973 | 77.7 14.0 8.3 3.1 25.4 56.5 | -0.144 | exp/ihm/nnet3/tdnn_sp/decode_eval/ascore_11/eval_hires.ctm.filt.sys
+
+CUED-RNNLM with CE objf:
+%WER 23.4 | 13098 94468 | 79.6 11.6 8.8 3.1 23.4 56.6 | -0.189 | exp/ihm/nnet3/tdnn_sp/decode_dev.rnnlm.ce.cued.lat.4gram/ascore_12/dev_hires.ctm.filt.sys
+%WER 24.5 | 12643 89977 | 78.4 13.4 8.2 2.9 24.5 55.6 | -0.164 | exp/ihm/nnet3/tdnn_sp/decode_eval.rnnlm.ce.cued.lat.4gram/ascore_11/eval_hires.ctm.filt.sys
+
+Mikolov-RNNLM:
+%WER 23.9 | 13098 94471 | 79.4 12.0 8.6 3.3 23.9 57.1 | -0.188 | exp/ihm/nnet3/tdnn_sp/decode_dev.rnnlm.mik.lat.4gram/ascore_12/dev.ctm.filt.sys
```

Comment:
Hmm the Mikolov script was not written by me (I think Guoguo wrote it) and I have done minimal tuning of CUED-RNNLM when comparing with Mikolov's. Also from the CUED-RNNLM paper, they reported of having around 0.5 - 0.7 gain over Mikolov's RNNLM when performing 50-best rescoring (Table 1) http://mi.eng.cam.ac.uk/~xl207/publications/conferences/ICASSP16-cued-rnnlm.pdf